src/main/java/module/training/ui/PlayerDetailPanel.java

```diff
@@ -311,22 +323,9 @@ private void selectTraining(ActionEvent e, PlayerSkill skillIndex) {
         }
     }
 
-    @Override
-    public void focusGained(FocusEvent e) {
-
-    }
-
-    @Override
-    public void focusLost(FocusEvent e) {
-        saveNotes();
-    }
-
-    private void saveNotes() {
-        var player = this.model.getActivePlayer();
-        if( player!= null){
-            player.setNote(m_jtaNotes.getText());
+    private void saveNotes(String notes) {
```

Comment:
Does an event need to get triggered to ensure the squad tables get refreshed, and displays the note?

Reply:
@tychobrailleur it was not as easy as expected....

Reply:
Sorry :-(